modules/ROOT/pages/logs-us-eu.adoc

```diff
@@ -0,0 +1,391 @@
+= Using Logs in Anypoint Monitoring (EU and US)
```

Comment:
No need to review **modules/ROOT/pages/logs-us-eu.adoc**: Just clarifying this applies only to US & EU. No plans to do improvements to this page at this time.